Makefile

```diff
@@ -91,6 +91,9 @@ pylint:
 	@pylint --disable=W0511,C0302,R0912,W1514,R1710 -s n csi/volumeutils.py
 	@pylint --disable=W0511,C0302,W1514 -s n operator/main.py
 	@pylint --disable=W0511,W1514 -s n extras/scripts/gen_manifest.py
+	@pylint --disable=W0511,C0301,C0103,R0903,R0914,C0201,E0401 -s n operator/exporter.py
+	@pylint --disable=W0511,C0301,C0103,R0914,E0401,C0114 -s n csi/exporter.py
+	@pylint --disable=W0511,C0103,E0401,C0114 -s n server/exporter.py
```

Comment:
- disable statements seems to be on a higher side, can you just comment (just the keywords of these codes) on what are the new ones?

Reply:
C0103: Invalid name style "app" 
R0903: Too few public methods
R0914: Too many locals
C0201: Consider iterating instead of .keys() [ This can be updated in separate PR ]
E0401: False positive "import errors" { which is clearly false since the program is running fine ]
C0114: Missing docstring at line 1.0 [ Looks like false positive again ] 
C0301: Line too long[ Can be fixed ]